Contributor-dashboard.md

```diff
@@ -0,0 +1,44 @@
+# Contributor dashboard
+
+## Overview
+The [contributor dashboard page](https://www.oppia.org/contributor-dashboard) on the Oppia site allows users to submit content suggestions (currently translations and practice questions) directly to lessons. These suggestions are then reviewed, and either accepted, or sent back for revision. See the [user docs](https://oppia-user-guide.readthedocs.io/en/latest/contributor/contribute.html) for step-by-step instructions how to contribute content suggestions.
+
+## How items for contribution are populated
+
+### Translate text tab
+Every list item on the "Translate Text" tab corresponds to a particular lesson and all of its pieces of text content. In the codebase, we refer to each of these list items as _opportunities_. The contributor dashboard automatically shows a translation opportunity for a lesson when the following are true:
+
+1. The lesson (also called an _exploration_) corresponds to a _chapter_ in a _story_ of a published classroom subject, e.g. "Decimals" (also called a _topic_). See the [user docs](https://oppia-user-guide.readthedocs.io/en/latest/keyconcepts.html) for an overview of these terms.
+1. Every piece of text content in the lesson does not yet have an accepted translation.
```

Comment:
This sounds like the entire lesson is untranslated. Maybe instead: "There is at least one piece of text content in the lesson that does not yet have an accepted translation." ?